boards/arm/96b_carbon_nrf51/dts.fixup

```diff
@@ -2,3 +2,5 @@
 #define CONFIG_UART_NRF5_IRQ_PRI	NORDIC_NRF_UART_40002000_IRQ_0_PRIORITY
 #define CONFIG_UART_NRF5_BAUD_RATE	NORDIC_NRF_UART_40002000_CURRENT_SPEED
 #define CONFIG_UART_NRF5_NAME		NORDIC_NRF_UART_40002000_LABEL
+
+#define CONFIG_SOC_FLASH_NRF5_DEV_NAME	NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL
```

Comment:
Can you also add to all the fixups:

#define FLASH_DEV_NAME CONFIG_SOC_FLASH_NRF5_DEV_NAME